pkg/timeutil/timeutil.go

```diff
@@ -20,9 +24,32 @@ func Date(datetime time.Time) time.Time {
 func LastWeekday(weekday time.Weekday) time.Time {
 	date := Date(time.Now())
 
-	for date.Weekday() != weekday {
+	for date.Weekday() !=  weekday{
 		date = date.AddDate(0, 0, -1)
 	}
 
 	return date
 }
+
+// StringToWeekday transforms a string to time.Weekday
+// it returns Monday if wrong day specified in config.toml
```

Comment:
This looks like it's broken shorter than at 100 columns, that's the standard I've gone for here.